slides/02-data-budget.qmd

```diff
@@ -267,26 +268,39 @@ frog_train %>%
 
 ```{r}
 #| fig-align: 'center'
-ggplot(frog_train, aes(reflex, age)) +
-  stat_summary_2d(aes(z = latency), 
-                  alpha = 0.7, binwidth = c(1, 5e3)) +
-  scale_fill_viridis_c() +
-  labs(fill = "mean latency")
+ggplot(frog_train, aes(age, latency, color = reflex)) +
+  geom_point(alpha = .8, size = 2)
```

Comment:
This is a very interesting plot. Why were the low/mid reflex frogs "jiggled" at such a young age? That doesn't feel like a very fair comparison VS the full reflex ones.
<img width="1176" alt="Screen Shot 2022-07-23 at 6 48 13 PM" src="https://user-images.githubusercontent.com/19150088/180625345-a95b4ed8-f077-4b57-a486-739a14f88f85.png">

Reply:
In the grand scheme of things that means they were jiggled approximately half a day earlier than the full reflex frogs

```
(400000 - 350000) / 86400 = .58 days
```

So maybe the lab tech did low/mid in the early morning, had lunch, and did the full reflex batch in the late afternoon 😆

Reply:
Do we want to divide the `age` column by 86400 in that preprocessing code? I can do that in a separate PR; I think that would make insights like the above easier to explain (it's one less thing to have to keep in mind)!

Reply:
If you have time to do that, great! But I don't think this is super high priority.